src/dispatch/alembic/versions/b6da2dad0396_.py

```diff
@@ -0,0 +1,45 @@
+"""Adds column for FAIR form of loss category
+
+Revision ID: b6da2dad0396
+Revises: a769ea6bad64
+Create Date: 2021-03-19 13:07:36.153462
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+from dispatch.database import SessionLocal
+from dispatch.incident_cost_type.models import IncidentCostTypeUpdate
+from dispatch.incident_cost_type import service as incident_cost_type_service
+
+
+# revision identifiers, used by Alembic.
+revision = "b6da2dad0396"
+down_revision = "a769ea6bad64"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
Would it be easier to just set a default for this column?

Reply:
I thought about adding a default, but decided not to to motivate users to think under which FAIR FoL phase they need to bucket the costs types they add.